languages/fi.ini

```diff
@@ -159,7 +159,7 @@ By Recent = "Uutuuden mukaan"
 By Region = "Alueen mukaan"
 By Title = "Nimekkeen mukaan"
 By Topic = "Aiheen mukaan"
-Call Number = "Sijainti"
+Call Number = "Hyllypaikka"
```

Comment:
Is this a generic change that should be applied to existing translations, or is it somehow specific to this PR?

Reply:
Specific to status display in Finnish, there were two columns with the same translation (this was being wrong)...

Reply:
Should I port this change over to the release-6.0 branch, then?

Reply:
That'd make sense. Thanks!

Reply:
One last question before I apply the change: I notice that the language file still contains the line `By Call Number = "Sijainnin mukaan"` -- should this be adjusted to match?

Reply:
...also `Multiple Call Numbers = "Useita sijainteja"`

Reply:
Maybe it's worth opening a separate PR to correct these three lines? Or you can just give me translations here and I'll fix it for you -- whatever is less work for you is fine with me!

Reply:
I think those are fine. "Sijainti" means "Location", and that's sort of shelf location.

Reply:
Okay, thanks for the confirmation. I've taken care of the merging of the fix!